packages/office-addin-dev-settings/src/sideload.ts

```diff
@@ -199,12 +201,32 @@ function getWebExtensionPath(
 }
 
 function isSideloadingSupportedForDesktopHost(app: OfficeApp): boolean {
-  if (app === OfficeApp.Outlook || app === OfficeApp.Project || app === OfficeApp.OneNote) {
+  if (app === OfficeApp.Outlook && process.platform !== "win32" || app === OfficeApp.Project || app === OfficeApp.OneNote) {
     return false;
   }
   return true;
 }
 
+function isSideloadingSupportedForWebHost(app: OfficeApp): boolean {
+  if (app === OfficeApp.Outlook || app === OfficeApp.Project) {
```

Comment:
I also think it would be better to have the logic reflect the apps that do support sideload and return true for them, or fallback to returning false at the end.